doc/contributors/releasegovernance.md

```diff
@@ -0,0 +1,46 @@
+# Release Governance
+
+The goal of this document is to define the method and schedule of Operator Lifecycle Manager releases.
+
+## Scope
+
+- Define a consistent release cadence for the project
+- Provide a framework to ensure consistent high quality releases
+- Define a consistent understanding of what is in scope for a release
+- Ensure community members understand the release cycle
+
+## Roles and Organization
+
+The Operator Lifecycle Manager project is part of the Operator Framework. As a result, it's community membership guidelines adhere to the definitions defined in the Operator Framework's [community membership definition](https://github.com/operator-framework/community/blob/master/community-membership.md). For the purpose of this document, there are two roles that are relevant for the release schedule.
+
+- Project reviewers provide guidance on whether or not a release is ready. Before a release can be cut, we will require that two reviewers provide the `/lgtm` label to the release.
+- Project approvers provide the final approval on whether or not a release is ready to be cut. A quorum (greater than 50%) of the project's approvers need to agree that a release is ready by writing the `/approve` label to the release.
+
+The set of reviewers and approvers is defined by the project's [OWNERS file](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/OWNERS)
+
+## Release Process
+
+### Minor Releases
+When a minor release is ready to be generated, a github issue will be filed against the project containing preliminary release notes. At that time, it should be labeled with the milestone expected for that release. Once that issue is opened, it needs approval from two reviewers and a majority of project approvers before the release can be scheduled. Additionally, at the same time, an email will be sent out to the operator framework mailing list with a link to the release notes in the issue. At that time, there is a minimum of a 48 hour comment period before the release can be completed.
+
```

Comment:
What's the value of the 48 hour comment period besides just slowing down the release process once its been approved? I can understand sending out the email but cutting a new release seems like safe operation, not potentially contentious like an enhancement, is there really a freeze period required?

Reply:
Will commenters be able to argue against cutting a release in the comment period? I think it makes sense to have some time for everyone to review the release before it goes out, but that I think that should be before the github issue gets approved.